docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,470 @@
+    To handle the received request, define and register an event handler for the `RequestReceived` event of the `Tizen.Applications.ComponentBased.ComponentPort` class as follows:
+
+    ```
+    {
+        _port = new ComponentPort("ServerService");
+        _port.RequestReceived += OnRequestReceived;
+        _port.AddPrivilege("http://tizen.org/privilege/datasharing");
+        _thread = new Thread(new ThreadStart(OnThread));
+        _thread.Start();
+    }
+
+    private void OnThread()
+    {
+        Log.Info("OnTrhead()");
+        _port.WaitForEvent();
+    }
+
+    private void OnRequestedReceived(object sender, RequestEventArgs args)
+    {
+        Log.Info("Sender: + args.Sender);
+        var request = args.Request;
+        if (request.GetType() == typeof(Request))
+        {
+            var req = (Request)request;
+            Log.Info("Command: " + req.Command);
+            Log.Info("Sequence: " + req.Sequence);
+            Log.Info("Message: " + req.Message);
+
+            if (args.IsReplyRequested)
+            {
+                args.Reply = new Response(req.Command, req.Sequence, 0);
+            }
+        }
+    }
+    ```
+
+3.  Send the request in the sending component as follows:
+
+    1. Call `WaitForEvent()` of the `Tizen.Applications.ComponentBased.ComponentPort` class in the thread, if you want to receive the reply from the **ServerService**.
+
+    2. Call `WaitForPort`() of the `Tizen.Applications.ComponentBased.ComponentPort` class to wait until the target port is ready.
+
+    3. Use `Send()` of the `Tizen.Applications.ComponentBased.ComponentPort` class to send the request.
+
+    4. Provide the request to be sent as an instance of the `ClientService.Tizen.Request` class as follows:
+
+    ```
+    {
+        _port = new ComponentPort("ClientService");
+        _port.RequestReceived += OnRequestReceived;
+        _thread = new Thread(new ThreadStart(OnThread));
+        _thread.Start();
+        ...
+        await ComponentPort.WaitForPort("ServerService");
+
+        try
+        {
+            var request = new Request("TestCommand", _seq++, "TestMessage");
+            _port.Send("ServerService", 5000, request);
+        }
+        catch
+        {
+            Log.Error("Failed to send request");
+        }
+        ...
+    }
+
+    private void OnThread()
+    {
+        Log.Info("OnTrhead()");
+        _port.WaitForEvent();
+    }
+
+    private void OnRequestedReceived(object sender, RequestEventArgs args)
+    {
+        Log.Info("Sender: + args.Sender);
+        var response = args.Request;
+        if (response.GetType() == typeof(Response))
+        {
+            var req = (Response)request;
+            Log.Info("Command: " + req.Command);
+            Log.Info("Sequence: " + req.Sequence);
+            Log.Info("Result: " + req.Result);
+        }
+    }
+    ```
+
+<a name="task"></a>
+## Use component task
```

Comment:
Just for consistency across the pages suggesting this:
```suggestion
## Using component task

Reply:
Done